sentry-core/src/session.rs

```diff
@@ -114,8 +114,109 @@ impl Session {
     }
 }
 
+// as defined here: https://develop.sentry.dev/sdk/envelopes/#size-limits
+const MAX_SESSION_ITEMS: usize = 100;
+const FLUSH_INTERVAL: u64 = 60;
```

Comment:
_nit_: You could specify this as `Duration::from_secs` directly to avoid any misunderstanding.